tests/order/matching_orders_02.ts

```diff
@@ -255,6 +255,132 @@ describe("Order Matching: Cross Liquidity", () => {
       [[1.009563, 0.994375, 0.995, 0.001062], 3],
     );
   });
+
+  it("after cancellation", async () => {
```

Comment:
this test is failing for me, I think ` "liquiditiesAgainst": Array [],` is returning empty

Reply:
yeah, this happens after I've added the code to cancel method that updates the cross match liquidity after removal